### PR TITLE
New package: sisl-4.6.0

### PR DESCRIPTION
**Pull request by winks**

### CODE REVIEW

**Comment by winks** [comment]
```
xbps-query -f sisl
/usr/include/sisl-copyright.h
/usr/include/sisl.h
/usr/include/sislP.h
/usr/lib/libsisl.a
/usr/share/licenses/sisl/COPYING
```

File: srcpkgs/sisl/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'sisl'
+pkgname=sisl
+version=4.6.0
+revision=1
+wrksrc="SISL-SISL-${version}"
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release"
```
Comment (jnbr) [inline srcpkgs/sisl/template]: this not needed, it's already set by the cmake build style

Comment (winks) [reply srcpkgs/sisl/template]: I'll check, but the INSTALL of that lib calls for `ccmake` where you select a release type, I found that a bit odd anyway.

Comment (winks) [reply srcpkgs/sisl/template]: You are right, it's on by default, I removed it.

**Pull request merged**